src/main/java/net/snowflake/ingest/streaming/internal/ArrowRowBuffer.java

```diff
@@ -556,16 +556,21 @@ private float convertRowToArrow(
             switch (physicalType) {
               case SB8:
                 {
+                  boolean ignoreTimezone = logicalType == ColumnLogicalType.TIMESTAMP_NTZ;
```

Comment:
nit: seems like this could be move out of the switch?

Reply:
Done